source/sass/mixins/_typography.scss

```diff
@@ -120,5 +120,29 @@
 @mixin list-style-none() {
   @include padding(0, inline-start);
   // Overflow auto is so the bullet will be still read from a screen reader and will push the bullet off screen
-  overflow: auto;
+  overflow-x: auto;
+}
+
+@mixin list-separator($separator: default) {
+  @if $separator == default {
+    &:after {
+      content: ",\a0";
+    }
+
+    &:lang(ar):after {
+      content: "،\a0";
+    }
+
+    &:lang(ja):after {
+      content: "、";
+    }
+  } @else if $separator == alternative {
+    html[lang][lang] &:after {
+      content: ";\a0";
```

Comment:
Doesn't reverse for RTL.

Reply:
This is what I see (both Chrome & FF):

<img width="105" alt="Screenshot 2019-06-19 14 21 53" src="https://user-images.githubusercontent.com/2893480/59769232-9f33be80-929d-11e9-8c77-90a95e25e1e5.png">

Should it be different?

Reply:
Would expect it to be flipped, but reading https://en.wikipedia.org/wiki/Semicolon#Arabic there's a dedicated character that we should add in. (No idea about other RTL languages.)